multi-event-scf-nodejs/serverless.template.yml

```diff
@@ -0,0 +1,20 @@
+
+name: multi-event-multi-event-scf-nodejs # 项目模板的名字 
```

Comment:
As this will be a featured template, this name loos a bit too long. Maybe we can call it `multi-scf-starter` ? So that users can init it by `sls init multi-scf-starter` cc @ole3021

Reply:
We should keep the template name in the same convention.  suggest

* call `multi-scf-nodejs`, `multi-scf-python` for default multi-scf use cases. (event is the default, and no need to mention this in the template name.)
* (for web function cases) call `scf-web-express` `multi-scf-web-django`  etc. 

@yugasun  cc: @AprilJC  ^

Reply:
这里名称是全局替换错误，我修复下